### PR TITLE
fixed incorrect markdown on some posts

### PR DESCRIPTION
**Pull request by shu8**
Removed extra spaces that caused the italics/bold to not be rendered.

### CODE REVIEW

**Comment by NickCraver** [comment]
Unfortunately this conflicted with another PR, can you please rebase so I can pull these fixes in?

**Comment by shu8** [comment]
@NickCraver sorry - I'm new to GitHub - what does `rebase` mean!?

**Comment by haneytron** [comment]
@shu8 `rebase` is a Git command. It allows you to "sync" your local branch with the remote branch, while maintaining your changes. Google it for some info.

**Comment by shu8** [comment]
@ironyx thanks for the response but I have no idea what to. I'm following https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request.

`git rebase seblog/master` results in:

> 

Sorry if I'm being stupid - I am _completely_ new to GitHub and git especially!

**Comment by haneytron** [comment]
@shu8 you're definitely not stupid - we all start somewhere and Git isn't exactly intuitive to use or user friendly ;). Hmm, I'm not super familiar with posh~git. Basically what that window above says is `CONFLICT (content): Merge conflict in disqus_import.py` which means that we changed `disqus_import.py` and you changed it also, and Git doesn't know how to correctly merge it. It is pausing here for you to fix the merge issue, at which point you would run `git rebase --continue` to continue with the rebase. Do you have a diff-merge tool? Can @NickCraver or someone else from our org recommend a good one? I use Tortoise-Git merge tools, which aren't great. Does this help?

Basically you need to do the following:
- Install a merge-diff tool
- Merge the `disqus_import.py` file that has a conflict
- Once merged, run `git rebase --continue` to proceed

-Haney

**Comment by shu8** [comment]
@ironyx thanks for the prompt reply! I'll wait to see if @NickCraver can suggest anything better, if not, I'll install Tortoise-Git merge tools and see how that works tomorrow :)

Thanks again for understanding my noob level ;)

**Comment by haneytron** [comment]
@shu8 anytime, and thanks for contributing to the blog project!

**Comment by JonHMChan** [comment]
Going to resolve this outside of a PR.
